### PR TITLE
Fix typo in cassandra-rabbitmq Dockerfile conf

### PR DESCRIPTION
**Pull request by Aschen**
Just a very small typo that I spotted by looking over the shoulder of @chibenwa at OpenTechSummit ;-)

### CODE REVIEW

**Comment by chibenwa** [comment]
Thanks Adrien for the fix!

For your information, I believe this typo is in other `extensions.properties` files that can be found in `dockerfiles/run/guice` path. 

Would it be possible for you to fix the typo there as well?

Benoit

**Comment by Aschen** [comment]
I also fix this typo in the code.

How can I run the test suite to check if I didn't broke something?

File: mailbox/tools/copier/src/test/java/org/apache/james/mailbox/tools/copier/MailboxCopierTest.java [hunk 1]
```diff
@@ -73,66 +73,66 @@ public void setup() throws BadCredentialsException, MailboxException {
...
         }
-        
+
         mailboxManager.endProcessingRequest(mailboxSession);
         mailboxManager.logout(mailboxSession, true);
-        
+
     }
-    
+
```
Comment (chibenwa) [inline mailbox/tools/copier/src/test/java/org/apache/james/mailbox/tools/copier/MailboxCopierTest.java]: Careful about unintendeed re-indents that pollutes the changeset.

We tend to put them in separated commits to ease reviewer job.

**Comment by chibenwa** [comment]
> How can I run the test suite to check if I didn't broke something?

You can run a `mvn clean install`. But this might take some time (dockerized integration test).

Let me play it against Linagora CI...

File: mailbox/tools/copier/src/test/java/org/apache/james/mailbox/tools/copier/MailboxCopierTest.java (hunk 1 above)
Comment (Aschen) [reply mailbox/tools/copier/src/test/java/org/apache/james/mailbox/tools/copier/MailboxCopierTest.java]: It was not a re-indent but only an automatic space cleaning, I could remove them if you want

**Comment by Aschen** [comment]
Looks like I didn't broke anything :D https://github.com/linagora/james-project/pull/2761

**Comment by chibenwa** [comment]
Hi,

This pull request has just been merged.

Thank you again for your dedication toward the James project. If you need help, want to [contribute](https://issues.apache.org/jira/issues/?jql=project%20%3D%20JAMES%20AND%20resolution%20%3D%20Unresolved%20AND%20labels%20%3D%20newbie%20ORDER%20BY%20priority%20DESC%2C%20updated%20DESC) [documentation](https://issues.apache.org/jira/issues/?jql=project%20%3D%20JAMES%20AND%20resolution%20%3D%20Unresolved%20AND%20labels%20%3D%20documentation%20ORDER%20BY%20priority%20DESC%2C%20updated%20DESC), new [features](https://issues.apache.org/jira/issues/?jql=project%20%3D%20JAMES%20AND%20resolution%20%3D%20Unresolved%20AND%20labels%20%3D%20feature%20ORDER%20BY%20priority%20DESC%2C%20updated%20DESC), tests or [bugfix](https://issues.apache.org/jira/issues/?jql=project%20%3D%20JAMES%20AND%20resolution%20%3D%20Unresolved%20AND%20labels%20%3D%20easyfix%20ORDER%20BY%20priority%20DESC%2C%20updated%20DESC), please don't hesitate to reach us.

Thanks again,

Cheers,

Benoit

**Pull request merged**